src/openrct2/scenes/title/TitleScene.cpp

```diff
@@ -53,15 +53,15 @@ bool TitleScene::PreviewSequence(size_t value)
     _previewingSequence = TryLoadSequence(true);
     if (_previewingSequence)
     {
-        if (!(gScreenFlags & SCREEN_FLAGS_TITLE_DEMO))
+        if (gLegacyScene != LegacyScene::titleSequence)
```

Comment:
These kinds of things may make it quite difficult to properly replace gScreenFlags/gScreenMode/gLegacyScreen.

Reply:
Not sure what you mean. The fact that it's called a `titleSequence` instead of a `titleScene`? I think that's fine, personally.

Reply:
No, what I meant was that, for example, TitleScene may be active, even when gLegacyScene is `playing`. So merging the two, as you seem to intend, may prove difficult. (This has no bearing on the current PR, it is just something that stood me out.)